tests/e2e/specs/dashboard/dashboard-noscript.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { setSiteVerification, setSearchConsoleProperty } from '../../utils';
+
+describe( 'Site Kit noscript message', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await setSiteVerification();
+	} );
+
+	beforeEach( async () => {
+		page.setJavaScriptEnabled( false );
```

Comment:
This should only be necessary `beforeAll` as I don't believe it will be reset between. It should also be reset to `true` `afterAll`.